tests/std/tests/P0466R5_layout_compatibility_and_pointer_interconvertibility_traits/test.cpp

```diff
@@ -0,0 +1,215 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+#include <assert.h>
+#include <type_traits>
+
+using namespace std;
+
+#define ASSERT(...) assert((__VA_ARGS__))
+
+
+constexpr bool test() {
+#ifndef __clang__ // TRANSITION, LLVM-Bug #: To Be Filed
+    // is_layout_compatible tests
+    {
+        struct S0 {
+            int v1;
+            int v2;
+        };
+
+        struct S1 {
+            S0 s1;
+            int v3;
+        };
+
+        struct S2 {
+            S0 s1;
+            int v2;
+        };
+
+        struct S3 {
+            S0 s1;
+            int v2;
+            int v3;
+        };
+
+        struct S4 {
+            int v1;
+
+        private:
+            int v2;
+        };
+
+        struct S5 {
+            int v1;
+
+        private:
+            int v2;
+        };
+
+
+        const struct S6 {

```

Comment:
`const` is meaningless here. Ditto for `volatile` below.

```suggestion
        using S6 = const struct {
```

Reply:
Thanks for pointing this out. I'll use this style in the tests:
ASSERT(is_layout_compatible_v<_const_ S1, S2>);

Reply:
Fixed